README.md

```diff
@@ -1,6 +1,9 @@
-# debugtopus  🐙💻
+# debugtopus  🐙 💻
+
+Hey!  Debugtopus is a tool to run Octomind tests against the local instance of your app. Octomind tests are AI auto-generated, hosted and run end-to-end tests in standard Playwright code in the cloud. They work on both staging and production, as long as we can access the website. 
+
+Here some [info](https://www.octomind.dev/?utm_source=github&utm_medium=debugtopus&utm_campaign=rm) on that and more in the [docs](https://www.octomind.dev/docs?utm_source=github&utm_medium=debugtopus&utm_campaign=rm).
```

Comment:
```suggestion
Here's some more [info](https://www.octomind.dev/?utm_source=github&utm_medium=debugtopus&utm_campaign=rm) on octomind and more in the [docs](https://www.octomind.dev/docs?utm_source=github&utm_medium=debugtopus&utm_campaign=rm).
```

Reply:
Im not sure if UTM-links in readme are good style, devs are finicky like that. But also fine I guess if you need it.